ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,272 @@
+    foverlay.suptitle(title, fontsize=24)
+    foverlay.set_size_inches(12.5, 10.5, forward=True)
+    return foverlay
+
+def get_true_volume(nparray):
+    """
+    because there is some spacing out side of brain, instead of simply choosing
+    1/4  1/2  3/4 part of all space to slide,
+    this function is first detect where is brain and then get 1/4 1/2 3/4 part of brain
+    """
+    img_arr = nparray.astype(int)
+    threshold = int(1)
+    img_arr[img_arr <= threshold] = 0
+    img_arr[img_arr > threshold] = 1
+    true_volume = np.where(img_arr == 1)
+    x = get_range(true_volume, 0)
+    y = get_range(true_volume, 1)
+    z = get_range(true_volume, 2)
+    return x, y, z
+
+def get_range(array,i):
+    min_num = min(array[i])
+    max_num = max(array[i])
+    arrange = np.arange(min_num, max_num)
+    quarter = np.percentile(arrange, [25, 50, 75]).astype(int)
+    return quarter
+
+def get_min_max(data, minthr=2, maxthr=95):
+    """
+    data: regbrain data to threshold.
+    """
+    min_val = np.percentile(data, minthr)
+    max_val = np.percentile(data, maxthr)
+    return (min_val.astype(float), max_val.astype(float))
+
+def get_braindata(brain_file):
+    """Opens a brain data series for a mask, mri image, or atlas.
+    Returns a numpy.ndarray representation of a brain.
+    Parameters
+    ----------
+    brain_file : str, nibabel.nifti1.nifti1image, numpy.ndarray
+        an object to open the data for a brain. Can be a string (path to a brain file),
+        nibabel.nifti1.nifti1image, or a numpy.ndarray
+    Returns
+    -------
+    array
+        array of image data
+    Raises
+    ------
+    TypeError
+        Brain file is not an accepted format
+    """
+
+    if type(brain_file) is np.ndarray:  # if brain passed as matrix
+        braindata = brain_file
+    else:
+        if type(brain_file) is str or type(brain_file) is str:
+            brain = nb.load(str(brain_file))
+        elif type(brain_file) is nb.nifti1.Nifti1Image:
+            brain = brain_file
+        else:
+            raise TypeError(
+                "Brain file is type: {}".format(type(brain_file))
+                + "; accepted types are numpy.ndarray, "
+                "string, and nibabel.nifti1.Nifti1Image."
+            )
+        braindata = brain.get_data()
+    return braindata
+
+def pad_im(image,max_dim,pad_val):
+    """
+    Pads an image to be same dimensions as given max_dim
+    Parameters
+    -----------
+    image: 3-d RGB np array of image slice
+    max_dim: dimension to pad up to
+    pad_val: value to pad with
+    Returns
+    -----------
+    padded_image: 3-d RGB np array of image slice with padding
+    """
+    #pad only in first two dimensions not in rgb
+    pad_width = (((max_dim-image.shape[0])//2,(max_dim-image.shape[0])//2),((max_dim-image.shape[1])//2,(max_dim-image.shape[1])//2),((max_dim-image.shape[2])//2,(max_dim-image.shape[2])//2))
```

Comment:
Break this long line into two lines, general formatting rules advise against having lines that are too long

Reply:
fixed it in the latest version